[2025-07-25T01:22:46.037+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bis_pipeline_dag.kafka_to_bronze manual__2025-07-25T01:22:41.688719+00:00 [queued]>
[2025-07-25T01:22:46.042+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bis_pipeline_dag.kafka_to_bronze manual__2025-07-25T01:22:41.688719+00:00 [queued]>
[2025-07-25T01:22:46.042+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-07-25T01:22:46.078+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): kafka_to_bronze> on 2025-07-25 01:22:41.688719+00:00
[2025-07-25T01:22:46.081+0000] {standard_task_runner.py:60} INFO - Started process 308 to run task
[2025-07-25T01:22:46.083+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'bis_pipeline_dag', 'kafka_to_bronze', 'manual__2025-07-25T01:22:41.688719+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/bis_pipeline_dag.py', '--cfg-path', '/tmp/tmppxydzq85']
[2025-07-25T01:22:46.084+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask kafka_to_bronze
[2025-07-25T01:22:46.151+0000] {task_command.py:423} INFO - Running <TaskInstance: bis_pipeline_dag.kafka_to_bronze manual__2025-07-25T01:22:41.688719+00:00 [running]> on host 665ba5283d17
[2025-07-25T01:22:46.263+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bis_pipeline_dag' AIRFLOW_CTX_TASK_ID='kafka_to_bronze' AIRFLOW_CTX_EXECUTION_DATE='2025-07-25T01:22:41.688719+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-25T01:22:41.688719+00:00'
[2025-07-25T01:22:46.297+0000] {docker.py:359} INFO - Starting docker container from image bitnami/spark:3.3
[2025-07-25T01:22:49.050+0000] {docker.py:429} INFO - [38;5;6mspark [38;5;5m01:22:48.87 [0m[38;5;2mINFO [0m ==>
[2025-07-25T01:22:49.051+0000] {docker.py:429} INFO - [38;5;6mspark [38;5;5m01:22:48.87 [0m[38;5;2mINFO [0m ==> [1mWelcome to the Bitnami spark container[0m
[2025-07-25T01:22:49.051+0000] {docker.py:429} INFO - [38;5;6mspark [38;5;5m01:22:48.87 [0m[38;5;2mINFO [0m ==> Subscribe to project updates by watching [1mhttps://github.com/bitnami/containers[0m
[2025-07-25T01:22:49.051+0000] {docker.py:429} INFO - [38;5;6mspark [38;5;5m01:22:48.87 [0m[38;5;2mINFO [0m ==> Submit issues and feature requests at [1mhttps://github.com/bitnami/containers/issues[0m
[2025-07-25T01:22:49.051+0000] {docker.py:429} INFO - [38;5;6mspark [38;5;5m01:22:48.87 [0m[38;5;2mINFO [0m ==>
[2025-07-25T01:22:49.051+0000] {docker.py:429} INFO - 
[2025-07-25T01:22:57.136+0000] {docker.py:429} INFO - python3: can't open file '/app/bronze.py': [Errno 2] No such file or directory
[2025-07-25T01:22:57.148+0000] {docker.py:429} INFO - 25/07/25 01:22:57 INFO ShutdownHookManager: Shutdown hook called
[2025-07-25T01:22:57.149+0000] {docker.py:429} INFO - 25/07/25 01:22:57 INFO ShutdownHookManager: Deleting directory /tmp/spark-43a3f503-4565-4006-97ea-2e0453aba7b8
[2025-07-25T01:22:58.578+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 502, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 364, in _run_image
    return self._run_image_with_mounts([*self.mounts, tmp_mount], add_tmp_variable=True)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/docker/operators/docker.py", line 437, in _run_image_with_mounts
    raise DockerContainerFailedException(f"Docker container failed: {result!r}", logs=log_lines)
airflow.providers.docker.exceptions.DockerContainerFailedException: Docker container failed: {'StatusCode': 2}
[2025-07-25T01:22:58.580+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=bis_pipeline_dag, task_id=kafka_to_bronze, execution_date=20250725T012241, start_date=20250725T012246, end_date=20250725T012258
[2025-07-25T01:22:58.637+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 18 for task kafka_to_bronze (Docker container failed: {'StatusCode': 2}; 308)
[2025-07-25T01:22:58.672+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-07-25T01:22:58.689+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
